src/components/sidemenu/SideMenuItem.js

```diff
@@ -35,11 +39,17 @@ const sideMenuItemStyles = ({ theme }) => ({
   },
   menuIcon: {
     marginLeft: theme.sizes.default,
-    marginRight: 20,
+    marginRight: 30,
+    width: 20,
+    textAlign: 'center',
   },
   menuText: {
     paddingRight: theme.sizes.defaultDouble,
   },
+  alignBottom: {
+    color: 'red',
```

Comment:
this seems to be  harcoding the red color for bottom items

Reply:
indeed, that was a mistake. Forgot to push changes, but it's removed